### PR TITLE
Feat/support wadors metadata

### PR DESCRIPTION
**Pull request by emelalkim**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
 node_modules
-build
+# build
```
Comment (emelalkim) [inline .gitignore:2]: we should remove build

File: src/aimHelper.js [hunk 2]
```diff
@@ -178,29 +178,32 @@ export function getAimImageData(image) {
   obj.image = [];
   const { aim, study, series, equipment, person } = obj;
 
-  aim.studyInstanceUid = image.data.string("x0020000d") || "";
+  console.log(" image in aim-api ===>");
```
Comment (emelalkim) [inline src/aimHelper.js:181]: clean consoles

File: src/aimHelper.js [hunk 3]
```diff
@@ -283,9 +286,11 @@ export function addUserToAimData({ name, loginName }, aimData) {
 }
 
 function getSingleImageData(image) {
+  console.log(" ====> image in aim api getSingleImageData");
```
Comment (emelalkim) [inline src/aimHelper.js:289]: clean consoles

Comment (OzgeYurtsever) [reply src/aimHelper.js:289]: done

**Pull request closed without merging**